Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
File "./theories/Reals/RIneq.v", line 786, characters 2-40:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/Ranalysis1.v", line 917, characters 2-51:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/Ranalysis4.v", line 369, characters 2-44:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/Raxioms.v", line 244, characters 2-9:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/Rgeom.v", line 192, characters 2-71:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./plugins/micromega/RMicromega.v", line 398, characters 2-9:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

Traceback (most recent call last):
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/bin/g2t-server-text", line 33, in <module>
    sys.exit(load_entry_point('graph2tac==0.1.0', 'console_scripts', 'g2t-server-text')())
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 259, in main
    initialize_loop(reader, s, textmode, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 230, in initialize_loop
    g = prediction_loop_text(r, s, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 152, in prediction_loop_text
    print(g)
  File "capnp/lib/capnp.pyx", line 1197, in capnp.lib.capnp._DynamicStructReader.__str__
capnp.lib.capnp.KjException: capnp/layout.c++:2265: failed: expected ref->kind() == WirePointer::LIST [0 == 1]; Message contains non-list pointer where list pointer was expected.
stack: 14ed2631129a 14ed2634870e 14ed26354c1a 14ed26355166 14ed2635581c 14ed26228e54 556adef628d1 556adf01101c 556adf019615 556adef34596 556adef2c8d3 556adef3d21e 556adef2c8d3 556adef3d21e 556adef2c8d3 556adef3d21e 556adef2c8d3 556adefdf731 556adefdf676 556adf013048 556adf00d963 556adee91122 556adf007c9e 556adf007862 556adf004a1e 556adefd2968 14ed279070b2 556adefd2870
File "./theories/Reals/RIneq.v", line 794, characters 2-15:
Error: Anomaly "Capnp protocol error 1"
Please report at http://coq.inria.fr/bugs/.

Traceback (most recent call last):
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/bin/g2t-server-text", line 33, in <module>
    sys.exit(load_entry_point('graph2tac==0.1.0', 'console_scripts', 'g2t-server-text')())
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 259, in main
    initialize_loop(reader, s, textmode, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 230, in initialize_loop
    g = prediction_loop_text(r, s, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 152, in prediction_loop_text
    print(g)
  File "capnp/lib/capnp.pyx", line 1197, in capnp.lib.capnp._DynamicStructReader.__str__
capnp.lib.capnp.KjException: capnp/layout.c++:2265: failed: expected ref->kind() == WirePointer::LIST [0 == 1]; Message contains non-list pointer where list pointer was expected.
stack: 15403329029a 1540332c770e 1540332d3c1a 1540332d4166 1540332d481c 1540331a7e54 5561a48da8d1 5561a498901c 5561a4991615 5561a48ac596 5561a48a48d3 5561a48b521e 5561a48a48d3 5561a48b521e 5561a48a48d3 5561a48b521e 5561a48a48d3 5561a4957731 5561a4957676 5561a498b048 5561a4985963 5561a4809122 5561a497fc9e 5561a497f862 5561a497ca1e 5561a494a968 1540348860b2 5561a494a870
File "./theories/Reals/Ranalysis1.v", line 1040, characters 2-9:
Error: Anomaly "Capnp protocol error 1"
Please report at http://coq.inria.fr/bugs/.

Traceback (most recent call last):
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/bin/g2t-server-text", line 33, in <module>
    sys.exit(load_entry_point('graph2tac==0.1.0', 'console_scripts', 'g2t-server-text')())
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 259, in main
    initialize_loop(reader, s, textmode, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 230, in initialize_loop
    g = prediction_loop_text(r, s, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 152, in prediction_loop_text
    print(g)
  File "capnp/lib/capnp.pyx", line 1197, in capnp.lib.capnp._DynamicStructReader.__str__
capnp.lib.capnp.KjException: capnp/layout.c++:2265: failed: expected ref->kind() == WirePointer::LIST [0 == 1]; Message contains non-list pointer where list pointer was expected.
stack: 14a505b4a29a 14a505b8170e 14a505b8dc1a 14a505b8e166 14a505b8e81c 14a505a61e54 563c4933b8d1 563c493ea01c 563c493f2615 563c4930d596 563c493058d3 563c4931621e 563c493058d3 563c4931621e 563c493058d3 563c4931621e 563c493058d3 563c493b8731 563c493b8676 563c493ec048 563c493e6963 563c4926a122 563c493e0c9e 563c493e0862 563c493dda1e 563c493ab968 14a5071400b2 563c493ab870
File "./theories/Reals/Ranalysis4.v", line 374, characters 2-44:
Error: Anomaly "Capnp protocol error 1"
Please report at http://coq.inria.fr/bugs/.

Traceback (most recent call last):
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/bin/g2t-server-text", line 33, in <module>
    sys.exit(load_entry_point('graph2tac==0.1.0', 'console_scripts', 'g2t-server-text')())
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 259, in main
    initialize_loop(reader, s, textmode, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 230, in initialize_loop
    g = prediction_loop_text(r, s, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 152, in prediction_loop_text
    print(g)
  File "capnp/lib/capnp.pyx", line 1197, in capnp.lib.capnp._DynamicStructReader.__str__
capnp.lib.capnp.KjException: capnp/layout.c++:2265: failed: expected ref->kind() == WirePointer::LIST [0 == 1]; Message contains non-list pointer where list pointer was expected.
stack: 1497dcdd329a 1497dce0a70e 1497dce16c1a 1497dce17166 1497dce1781c 1497dcceae54 558d83fb18d1 558d8406001c 558d84068615 558d83f83596 558d83f7b8d3 558d83f8c21e 558d83f7b8d3 558d83f8c21e 558d83f7b8d3 558d83f8c21e 558d83f7b8d3 558d8402e731 558d8402e676 558d84062048 558d8405c963 558d83ee0122 558d84056c9e 558d84056862 558d84053a1e 558d84021968 1497de3c90b2 558d84021870
File "./theories/Reals/Rgeom.v", line 201, characters 2-71:
Error: Anomaly "Capnp protocol error 1"
Please report at http://coq.inria.fr/bugs/.

Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
File "./theories/Reals/ConstructiveCauchyRealsMult.v", line 575, characters 2-24:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/ConstructiveCauchyReals.v", line 534, characters 2-9:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/Ranalysis4.v", line 387, characters 2-30:
Error:
Anomaly "Uncaught exception Unix.Unix_error(Unix.EPIPE, "single_write", "")."
Please report at http://coq.inria.fr/bugs/.

File "./theories/Reals/RIneq.v", line 1622, characters 2-45:
Error:
Anomaly "Uncaught exception Unix.Unix_error(Unix.EPIPE, "single_write", "")."
Please report at http://coq.inria.fr/bugs/.

Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
File "./theories/QArith/Qround.v", line 73, characters 0-18:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

File "./plugins/setoid_ring/Ncring_initial.v", line 32, characters 7-24:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

Traceback (most recent call last):
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/bin/g2t-server-text", line 33, in <module>
    sys.exit(load_entry_point('graph2tac==0.1.0', 'console_scripts', 'g2t-server-text')())
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 259, in main
    initialize_loop(reader, s, textmode, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 230, in initialize_loop
    g = prediction_loop_text(r, s, tokenizer, model)
  File "/raid/scratch/piepejel/projects/coq-bench-15may/bd/target-source/venv/lib/python3.10/site-packages/graph2tac/transformer/pserver.py", line 152, in prediction_loop_text
    print(g)
  File "capnp/lib/capnp.pyx", line 1197, in capnp.lib.capnp._DynamicStructReader.__str__
capnp.lib.capnp.KjException: capnp/layout.c++:2265: failed: expected ref->kind() == WirePointer::LIST [0 == 1]; Message contains non-list pointer where list pointer was expected.
stack: 147c709c329a 147c709fa70e 147c70a06c1a 147c70a07166 147c70a0781c 147c708dae54 56199a7978d1 56199a84601c 56199a84e615 56199a769596 56199a7618d3 56199a77221e 56199a7618d3 56199a77221e 56199a7618d3 56199a77221e 56199a7618d3 56199a814731 56199a814676 56199a848048 56199a842963 56199a6c6122 56199a83cc9e 56199a83c862 56199a839a1e 56199a807968 147c71fb90b2 56199a807870
File "./theories/Reals/ConstructiveCauchyReals.v", line 768, characters 2-9:
Error: Anomaly "Capnp protocol error 1"
Please report at http://coq.inria.fr/bugs/.

Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
File "./plugins/micromega/QMicromega.v", line 128, characters 2-9:
Error: Anomaly "Capnp protocol error 4"
Please report at http://coq.inria.fr/bugs/.

